docs/REPRODUCIBLE.md

```diff
@@ -0,0 +1,83 @@
+
+# Reproducible Builds
+
+This section describes how to rebuild Space ROS and run the unit tests in the Docker container. Make sure the dev variant of the space ROS image is ready before proceeding.
+
+## Preparing Space ROS sources
+
+A manifest of the exact sources of Space ROS used to produce the current image is saved as `spaceros.repos` in the `/opt/spaceros` directory.
+To clone all sources from this manifest you can use the command sequence
+
+```bash
+spaceros-user@d10d85c68f0e:~/$ mkdir -p spaceros_ws/src
+spaceros-user@d10d85c68f0e:~/spaceros_ws$ cd spaceros_ws
+spaceros-user@d10d85c68f0e:~/spaceros_ws$ vcs import src < /opt/spaceros/scripts/spaceros.repos
```

Comment:
Just a general comment, but IMO this is going to be the real utility in what Space ROS could be. Namely, some base ROS 2 functionality with tools to make sure that what developers build comply with the standards those tools help verify. Sometime down the road, it would be nice if we could use these base images to push improvements back to the core ros repos we're building, and help make the ecosystem more robust overall while building towards a collection of packages that is "flight certifiable."

Just food for thought for when we're looking at how to get source code accessible...

Reply:
Agreed. I have opened an issue related to this here #211